app/src/main/java/com/github/gotify/MarkwonFactory.java

```diff
@@ -0,0 +1,77 @@
+package com.github.gotify;
+
+import android.content.Context;
+import android.graphics.Color;
+import android.graphics.Typeface;
+import android.text.style.BackgroundColorSpan;
+import android.text.style.BulletSpan;
+import android.text.style.QuoteSpan;
+import android.text.style.RelativeSizeSpan;
+import android.text.style.StyleSpan;
+import android.text.style.TypefaceSpan;
+
+import androidx.annotation.NonNull;
+
+import com.squareup.picasso.Picasso;
+
```

Comment:
> Is this an acceptable behaviour? As a fallback we can change it and only show the stripped message in the notification, which would look similar to this:

I think limited support is better than nothing. Can we remove the highlighting of links? That they don't look clickable. I already see the bug reports for this (:.

Reply:
Yes this might be a problem :) Links are shown as normal text now.

Reply:
This is oke.